tests/test_diffusers.py

```diff
@@ -663,61 +660,29 @@ def test_no_generation_regression_ldm3d(self):
             use_hpu_graphs=True,
             gaudi_config=GaudiConfig(),
         )
-        set_seed(27)
+
+        prompt = "An image of a squirrel in Picasso style"
+        generator = torch.manual_seed(seed)
         outputs = pipeline(
-            prompt="An image of a squirrel in Picasso style",
+            prompt=prompt,
+            generator=generator,
             output_type="np",
         )
 
         if os.environ.get("GAUDI2_CI", "0") == "1":
-            expected_slice_rgb = np.array(
-                [
-                    0.2099357,
-                    0.16664368,
-                    0.08352646,
-                    0.20643419,
-                    0.16748399,
-                    0.08781305,
-                    0.21379063,
-                    0.19943115,
-                    0.04389626,
-                ]
-            )
-            expected_slice_depth = np.array(
-                [
-                    0.68369114,
-                    0.6827824,
-                    0.6852779,
-                    0.6836072,
-                    0.6888298,
-                    0.6895473,
-                    0.6853674,
-                    0.67561126,
-                    0.660434,
-                ]
-            )
+            target_score = 29.63
         else:
-            expected_slice_rgb = np.array([0.7083766, 1.0, 1.0, 0.70610344, 0.9867363, 1.0, 0.7214538, 1.0, 1.0])
-            expected_slice_depth = np.array(
-                [
-                    0.919621,
-                    0.92072034,
-                    0.9184986,
-                    0.91994286,
-                    0.9242079,
-                    0.93387043,
-                    0.92345214,
-                    0.93558526,
-                    0.9223714,
-                ]
-            )
+            target_score = 35.81
```

Comment:
Bottomline is chosen as 95% of the mean of the CLIP scores I got from multiple runs (5-10 runs).